non-free.md

```diff
@@ -226,7 +226,7 @@
 
 **[`^        back to top        ^`](##awesome-selfhosted---non-free-software)**
 
-- [Deskpro](https://www.deskpro.com/) - On-Premise helpdesk software that includes email, chat, voice & helpcentre publishing. Full visible source code and API. `⊘ Proprietary` `Unknown`
+- [Deskpro](https://www.deskpro.com/) - On-Premise help desk software that includes email, chat, voice & help centre publishing. Full visible source code and API. `⊘ Proprietary` `Unknown`
```

Comment:
`helpdesk` (single word) looks correct https://dictionary.cambridge.org/dictionary/english/helpdesk but `help desk` is also correct https://dictionary.cambridge.org/dictionary/english/help-desk. We use both in https://github.com/awesome-selfhosted/awesome-selfhosted#readme. It's just a remark, I'm OK with this change.